libqtile/layout/max.py

```diff
@@ -31,6 +31,16 @@ class Max(_SimpleLayoutBase):
     commands to switch to next and previous windows in the stack.
     """
 
+    __defaults__ = [
+        ("margin", 0, "Margin of the layout (int or list of ints [N E S W])"),
+        ("border_color", "#0000ff", "Border colour(s) for the window"),
+        ("border_width", 1, "Border width."),
```

Comment:
Given that the code currently has `0` hard coded for the value, we should use that as the default here.